src/test/java/be/syntra/devshop/DevshopFront/TestUtils/JsonUtils.java

```diff
@@ -0,0 +1,10 @@
+package be.syntra.devshop.DevshopFront.TestUtils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class JsonUtils {
+
+    public static String returnObjectAsJsonString(Object object) throws Exception {
+        return new ObjectMapper().writeValueAsString(object);
```

Comment:
Objectmapper can be set up for dependency injection.
So you don't have to create new ObjectMapper. But objectmapper.writeValueAsString(object) instead.
example is used in the back-end tests.

Reply:
Fixed